Controller/Annotations/Route.php

```diff
@@ -123,7 +123,11 @@ public function __construct(
             );
         }
 
-        if (!$this->getMethods()) {
+        if ($this->isMethodsPropertyPublic()) {
```

Comment:
Curious why not use `if (isset($this->methods)) {` to handle this bc layer? Reflection looks like overhead.